openlibrary/macros/StarRatingsStats.html

```diff
@@ -6,7 +6,7 @@
 <ul class="readers-stats" itemprop="aggregateRating" itemscope itemtype="https://schema.org/AggregateRating">
   <li class="avg-ratings">
     $if rating_stats.get('avg_rating'):
-      $:('<span class="readers-stats__star">★</span>' * int(rating_stats.get('avg_rating'))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
+      $:('<span class="readers-stats__star">★</span>' * int((int(2 * float(rating_stats.get('avg_rating')))) - (int(rating_stats.get('avg_rating'))))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
```

Comment:
https://docs.python.org/3/library/functions.html#round

Reply:
@cclauss at first, I used round(), but I was getting this error
![image](https://user-images.githubusercontent.com/64412143/103780191-acb8c100-505a-11eb-8a6c-2f5c6d22081b.png)